### PR TITLE
Added mailers for a few more things, baseline for check sending

### PR DESCRIPTION
**Pull request by tmb**
- Added a mailer for when a transaction hits an events' account
- Added a mailer for when a G Suite account is created (notifying the user & creator)
- Added baseline service for Lob (check sending- no features, yet)

### CODE REVIEW

File: app/mailers/g_suite_account_mailer.rb [hunk 1]
```diff
@@ -15,6 +15,15 @@ def notify_user_of_activation(params)
       subject: 'Your Bank G Suite account is ready for you!'
   end
 
+  def notify_creator_of_activation(params)
+    @recipient = params[:recipient]
+    @first_name = params[:first_name]
+    @last_name = params[:last_name]
+
+    mail to: @recipient,
+      subject: "The Bank G Suite account that you created for #{@first_name} #{@last_name} has been activated."
```
Comment (lachlanjc) [inline app/mailers/g_suite_account_mailer.rb]:
```suggestion
      subject: "The Bank G Suite account you created for #{@first_name} #{@last_name} has been activated"
```

File: app/mailers/money_received_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class MoneyReceivedMailer < ApplicationMailer
+  def money_received
+    @transaction = params[:transaction]
+    @emails = @transaction.event.users.pluck(:email)
+    
+
```
Comment (lachlanjc) [inline app/mailers/money_received_mailer.rb]:
```suggestion
```

File: app/services/lob_service.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  def client
+    Lob::Client.new(
+      env: env,
+      client_id: client_id,
+      secret: secret_key,
+      public_key: public_key
+    )
+  end
+end
```
Comment (lachlanjc) [inline app/services/lob_service.rb]: End of line break should be included

File: app/views/g_suite_account_mailer/notify_creator_of_activation.html.erb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<p>Hi there,</p>
+
+<p>
+  <%= @first_name %> <%= @last_name %>'s Bank G Suite account has been sucessfully created and activated, and <%= @first_name %> has received an email with their username/password.
```
Comment (lachlanjc) [inline app/views/g_suite_account_mailer/notify_creator_of_activation.html.erb:4]: Rich quote

File: app/views/money_received_mailer/money_received.html.erb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<p>
+  Hi,
```
Comment (lachlanjc) [inline app/views/money_received_mailer/money_received.html.erb:2]: This greeting feels super weird, let's use people's names

File: app/services/lob_service.rb (hunk 3 above)
Comment (tmb) [inline app/services/lob_service.rb]:
```suggestion
end

```

File: app/views/money_received_mailer/money_received.html.erb (hunk 5 above)
Comment (tmb) [reply app/views/money_received_mailer/money_received.html.erb:2]: This is how we're currently doing it across the whole app, I'll open another PR to change it all at once.

**Comment by tmb** [comment]
This looks good for now, I want to make sure it makes it into the merge, so merging into `one`

**Pull request merged**